types/loadable__webpack-plugin/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for @loadable/webpack-plugin 5.2
+// Project: https://github.com/smooth-code/loadable-components
+// Definitions by: Martynas Kadiša <https://github.com/martynaskadisa>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { Plugin } from 'webpack';
+
+interface Options {
+	/**
+	 * Stats filename (default to loadable-stats.json)
+	 */
+	filename?: string;
+	/**
+	 * Always write assets to disk (default to false)
+	 */
+	writeToDisk?: boolean;
+}
+
+declare class LoadablePlugin extends Plugin {
+	constructor(options?: Options);
+}
+
+export default LoadablePlugin;
```

Comment:
This should be `export = LoadablePlugin`, given this example from this docs:
``js
const LoadablePlugin = require('@loadable/webpack-plugin')

module.exports = {
  plugins: [new LoadablePlugin()],
}
```
consumers who wish to use it as a default import should then specify `esModuleInterop: true` in their `tsc` options.

Reply:
I've looked at what's happening in [source code](https://github.com/smooth-code/loadable-components/blob/master/packages/webpack-plugin/src/index.js#L119-L120) and it seems it's done both ways, with `module.exports = ` and `module.exports.default = `. I only left `default export` since I'm not sure if it's a good idea or even possible to have `export = ` and `export default` at the same time in typings.

Reply:
You wanna use `export=` and merge a namespace with a default export with the class, then, so both are available.